projects/hslayers/common/dialogs/dialog-container.component.ts

```diff
@@ -17,24 +23,22 @@ export class HsDialogContainerComponent implements OnInit, OnDestroy {
   @ViewChild(HsDialogHostDirective, {static: true})
   dialogHost: HsDialogHostDirective;
   interval: any;
-  private end = new Subject<void>();
+  private destroyRef = inject(DestroyRef);
```

Comment:
Per the [Angular docs](https://angular.dev/api/core/rxjs-interop/takeUntilDestroyed),

_"optionally, the DestroyRef representing the current context. This can be passed explicitly to use takeUntilDestroyed outside of an [injection context](https://angular.dev/guide/di/dependency-injection-context). Otherwise, the current DestroyRef is injected."_

so.. can't we omit also the DestroyRef injection in most of the cases? I understand that it is only useful if we would like to react on DestroyRef of another component (like a cascade).

Reply:
That 'Otherwise, the current DestroyRef is injected.' part feels a bit missleading or I dont understand what they ment exactly because outside of the constructor it throws
![image](https://github.com/user-attachments/assets/c255c3d1-8c3c-45a4-90a1-7ac00f6f288a)

But I'll go through them once more as there are some cases where its not necessary like [here](https://github.com/hslayers/hslayers-ng/blob/73c253b31026943ca9be5f8906d3eb660657e4bd/projects/hslayers/components/styler/styler.component.ts#L45)
I was jsut on autopilot replacing those 'this.end's  😄

Reply:
[4745d99](https://github.com/hslayers/hslayers-ng/pull/5142/commits/4745d9936649e06d1034ad946b3ee950a2b57856)

Reply:
Oh, now the "injection context" is a bit clearer to me :D I haven't spot this limitation, thanks!